integration-cli/requirements.go

```diff
@@ -139,11 +139,11 @@ var (
 	}
 	NotGCCGO = testRequirement{
 		func() bool {
-			out, err := exec.Command("go", "version").Output()
-			if err != nil && strings.Contains(string(out), "gccgo") {
-				return true
+			out, _ := exec.Command("go", "version").Output()
```

Comment:
@runcom Actually checking err != nil should be added but due to issue with earlier logic, one of the tests is skipped even for golang and now the test is broken. Not sure if the intention is to skip the test. TestAppArmorDeniesPtrace is consistently failing on jenkins. This test passes in my local env.